contentctl/objects/risk_event.py

```diff
@@ -1,32 +1,51 @@
 import re
-from typing import Union, Optional
 
-from pydantic import BaseModel, Field, PrivateAttr, field_validator
+from pydantic import BaseModel, Field, PrivateAttr, field_validator, computed_field
 
 from contentctl.objects.errors import ValidationFailed
 from contentctl.objects.detection import Detection
 from contentctl.objects.observable import Observable
 
-# TODO (PEX-433): use SES_OBSERVABLE_TYPE_MAPPING
+# TODO (#259): Map our observable types to more than user/system
+# TODO (#247): centralize this mapping w/ usage of SES_OBSERVABLE_TYPE_MAPPING (see
+#   observable.py) and the ad hoc mapping made in detection_abstract.py (see the risk property func)
 TYPE_MAP: dict[str, list[str]] = {
-    "user": ["User"],
-    "system": ["Hostname", "IP Address", "Endpoint"],
-    "other": ["Process", "URL String", "Unknown", "Process Name"],
+    "system": [
+        "Hostname",
+        "IP Address",
+        "Endpoint"
+    ],
+    "user": [
+        "User",
+        "User Name",
+        "Email Address",
+        "Email"
+    ],
+    "hash_values": [],
+    "network_artifacts": [],
+    "host_artifacts": [],
+    "tools": [],
+    "other": [
+        "Process",
+        "URL String",
+        "Unknown",
+        "Process Name",
+        "MAC Address",
+        "File Name",
+        "File Hash",
+        "Resource UID",
+        "Uniform Resource Locator",
+        "File",
+        "Geo Location",
+        "Container",
+        "Registry Key",
+        "Registry Value",
+        "Other"
+    ]
```

Comment:
Is there a reason that we have not unified this object with the following, in one single place?
https://github.com/splunk/contentctl/pull/241/files#diff-9e1935fe7233e929a9fe50af50183ca0a44b882b79e198b8fbe09ee75c00b55dR332
(other than the fact that what is linked above is more legacy code)?

If these SHOULD be the same, then both parts of the code should refer to one, consistent object.

Reply:
We certainly can and should unify these and I have captured the intent to do so in an issue already: https://github.com/splunk/contentctl/issues/247

Doing so in this PR felt out of scope (trying to avoid scope creep)

Reply:
agreed that this is out of scope. resolved